### ISSUE TITLE
$HTTP_RAW_POST_DATA is deprecated

### ISSUE THREAD

**Issue by ed-fruty** [laravel/spark#81]
I installed spark on the fresh laravel project.
When I try to register user, js redirects me to the /undefined page.
I saw the response and it is like:

```
<br />
<b>Deprecated</b>:  Automatically populating $HTTP_RAW_POST_DATA is deprecated and will be removed in
 a future version. To avoid this warning set 'always_populate_raw_post_data' to '-1' in php.ini and use
 the php://input stream instead. in <b>Unknown</b> on line <b>0</b><br />
{"email":["The email must be a valid email address."],"password":["The password confirmation does not
 match."],"terms":["The terms must be accepted."]}
```

**Comment by aaranmcguire** [comment]
Whats your PHP version?

**Comment by ed-fruty** [comment]
```
vagrant@scotchbox:/var/www$ php -v
PHP 5.6.10-1+deb.sury.org~trusty+1 (cli) 
Copyright (c) 1997-2015 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2015 Zend Technologies
    with Zend OPcache v7.0.6-dev, Copyright (c) 1999-2015, by Zend Technologies
```

**Comment by ed-fruty** [comment]
Firstly I tried to run in by php-build server by `php artisan serve` and i got the same error.
After that, i configure vagrant (https://github.com/scotch-io/scotch-box ) and using them i still get this error.

**Comment by aaranmcguire** [comment]
I'm on `5.6.11` and I'm not getting that issue, can you gist up your php.ini and I will compare settings.

**Comment by ed-fruty** [comment]
Google give me link to the dialog with the developer, who added this feature to the `PHP 5.6`.
http://www.serverphorums.com/read.php?7,1048319,1048407

So solution realy is only set `always_populate_raw_post_data` to `-1`. (using `ini_set` for this option doesn't work).

I tried to grep this option

```
vagrant@scotchbox:/var/www$ grep -Hirn "always_populate_raw_post_data" /etc/php5/
/etc/php5/cli/php.ini:704:;always_populate_raw_post_data = -1
/etc/php5/cgi/php.ini:704:;always_populate_raw_post_data = -1
/etc/php5/apache2/php.ini:704:;always_populate_raw_post_data = -1
/etc/php5/fpm/php.ini:704:;always_populate_raw_post_data = -1
```

and founded, that it commented everywhere in `php.ini` files.
So, I uncomment 704 line in `/etc/php5/apache2/php.ini` and restart apache.
Now, always working correctly. 
Thanks.

**Comment by cwhite92** [comment]
I also get this warning on Laravel 5, I solved it by using this one-liner in my Vagrant box's provisioning script:

    sed -i 's/;always_populate_raw_post_data = -1/always_populate_raw_post_data = -1/' /etc/php5/apache2/php.ini

@ed-fruty that line should be particularly relevant to you as I was also using scotchbox.

Here is more info: http://php.net/manual/en/ini.core.php#ini.always-populate-raw-post-data

> This feature was DEPRECATED in PHP 5.6.0, and REMOVED as of PHP 7.0.0.

**Comment by phroggyy** [comment]
Yeah, your only option should be to change `always_populate_raw_post_data`. On most PHP installations, this will be the case from the beginning already.
